<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpMenuClass</class>
 <widget class="QMainWindow" name="SignUpMenuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignUpMenu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>300</width>
      <height>260</height>
     </rect>
    </property>
    <property name="title">
     <string>Create an account to climb to the top!</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username2LineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="password2LineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="signUp2Button">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>155</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="signUpErrorLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="leftImageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>110</width>
      <height>280</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rightImageLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>110</width>
      <height>280</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="back2PushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
